src/TMap.cpp

```diff
@@ -1532,7 +1532,7 @@ bool TMap::restore( QString location )
         }
     }
 
-    if( ! canRestore || entries.size() == 0 ) {
+    if( (! canRestore || entries.size() == 0) && downloadIfNotFound ) {
```

Comment:
💡  So the new `bool` argument to `TMap::restore(...)` has a default that maintains the existing behaviour but disables the auto-magic dialog to ask the use if they want to download a MUD provided map ___for IRE Muds only___ if they do not have a valid map already if it is provided and is `false`...